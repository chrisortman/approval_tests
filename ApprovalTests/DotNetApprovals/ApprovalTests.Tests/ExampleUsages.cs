using MbUnit.Framework;

namespace ApprovalTests.Tests
{
	[TestFixture]
	public class ExampleUsages
	{
		[Test, Parallelizable]
		public void ApprovingText()
		{
			Approvals.Approve("A piece of text");
		}

		[Test, Parallelizable]
		public void ApprovingAnEnumerble()
		{
			Approvals.Approve(new []{"item1", "item2", "item3"}, "items");
		}

		[Test, Parallelizable]
		public void ApprovingAnImage()
		{
		}

		[Test, Parallelizable]
		public void ApprovingAForm()
		{
		}

		[Test, Parallelizable]
		public void ApprovingAnObject()
		{
		}
	}
}