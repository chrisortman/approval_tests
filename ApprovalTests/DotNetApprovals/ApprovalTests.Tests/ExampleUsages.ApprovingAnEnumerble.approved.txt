items[0] = item1
items[1] = item2
items[2] = item3
